addons/web/static/src/libs/bootstrap/bootstrap.scss

```diff
@@ -0,0 +1,10 @@
+.form-control {
+    // Overridden file input buttons theming
+    &::file-selector-button {
+        @include gradient-bg(darken($input-bg, 2%));
+    }
+
+    &:hover:not(:disabled):not([readonly])::file-selector-button {
+        background-color: darken($input-bg, 6%);
+    }
+}
```

Comment:
We already have a file for this kind of things, called `bootstrap_review.scss`

Reply:
Thanks for drawing attention, will do the change